src/UI/templates/default/MessageBox/messagebox.less

```diff
@@ -0,0 +1,17 @@
+div.alert {
+	div {
+		margin-top: 10px;
+	}
+	ul {
+		margin-top: 15px;
+		background-color: @il-primary-container-bg;
+		padding: 10px 10px 10px 10px;
```

Comment:
why not working with @padding-large-vertical/horizontal here? are the design reasons? if so, consider introducing new variables for this, which depend on @padding-large-vertical/horizontal.